src/backend/routes/api.php

```diff
@@ -321,6 +321,7 @@ static function () {
 Route::group(['prefix' => 'export'], static function () {
     Route::get('/transactions/{slug}', 'TransactionExportController@download');
     Route::get('/debts/{slug}', 'OutstandingDebtsExportController@download');
+    Route::get('/customers/{slug}', 'PersonExportController@download');
```

Comment:
Could you explain what this `slug` is? In the collection above I see it's set to email of the user 🤔 Why is this needed? 
Looking at the other exports it "looks like" this was meant to be a way to pass a filter for the export, but it doesn't appear to be functional

Reply:
I found that the slug is used as an identifier, which is specifically an email (this is some way to generalize exporting records)

See https://github.com/EnAccess/micropowermanager/blob/360eb39783d5536c947060cd29aec1d3a15e347a/src/backend/app/modules/TenantResolver/ApiResolvers/DataExportResolver.php#L19

Reply:
Ufff 😅 Does this mean that anyone just send an HTTP call `/api/export/customers/demo_company_admin@example.com?format=csv` ? I.e. if I can "guess" a companies email, I can export their entire data from MPM?

Reply:
Oh that is correct at the moment 😱

The company ID should probably not be resolved by a user-given identifier.  Good catch